src/groupByKey.ts

```diff
@@ -2,6 +2,16 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+export function groupByKey(items:[], key: string): GroupsMap<object> {
```

Comment:
In general function has to take <T> generic, but your function do not. And this <T> is actually an array of items (items: T[ ])